<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pseudomonas</class>
 <widget class="QWidget" name="Pseudomonas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LeaveTypeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>420</width>
     <height>38</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>事假</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>病假</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>年假</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>婚假</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>65</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>产假</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>72</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>看护假</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>职员：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>工号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>部门：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>职位：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>时段：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>工时：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>工作转接：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>请假事由：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnOK">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>96</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QpbtnClose">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>96</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Number">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="departmentName">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="position">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Manhour">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Workfor">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Account">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="STime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="ETime">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>140</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
